public/app/plugins/panel/timeseries/utils.ts

```diff
@@ -33,6 +68,10 @@ export function prepareGraphableFields(
     series = prepareTimeSeriesLong(series);
   }
 
+  if (series.some((frame) => frame.fields.some((field) => field.type === FieldType.enum))) {
```

Comment:
```suggestion
  if (series.filter((frame) => frame.fields.some((field) => field.type === FieldType.enum)).length > 1) {
```
maybe? we only need to do this if there are >1 enum fields right

Reply:
yeah, i can change this to reduce()